src/js/common/helpers/api.js

```diff
@@ -21,8 +21,8 @@ export function apiRequest(resource, optionalSettings = {}, success, error) {
     }
   };
 
-  const settings = merge(defaultSettings, optionalSettings);
-
+  // if settings are complete, no need to merge with default
+  const settings = (complete ? optionalSettings :  merge(defaultSettings, optionalSettings));
```

Comment:
The default setting for X-Key-Inflection is not required for form removal, so it is not included.

Reply:
Does it break if it's included? I don't think it's bad practice to always send it.

Reply:
It breaks and is commented it out in [`removeInProgressForm`](https://github.com/department-of-veterans-affairs/vets-website/blob/0d8b592f954dda22ca9bfa8157ff7018bdd6c8b4/src/js/common/schemaform/save-load-actions.js#L339).

Reply:
Yeah, I remember that also being a problem in `fetchInProgressForm` because of back end Olive Branch reasons...I don't like this extra parameter to filter it out, though. Feels gross. 😕

Reply:
Should another version of this function be setup with different defaults? Then the other form related actions could also use it?